src/components/Home.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchApi } from './redux/previewSlice';
+import Header from './Header';
+import Preview from './Preview';
+import MainPreview from './MainPreview';
+import './stylesheets/Home.scss';
+
+function Home() {
+  const pokemons = useSelector((state) => state.preview);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!pokemons.length) dispatch(fetchApi());
+  });
+
+  return (
+    <>
+      <Header />
+      <MainPreview />
+      <h3>POKEMON BY ID</h3>
+      <section className="list">
+        {pokemons.map((pokemon) => (<Preview key={pokemon.id} pokemon={pokemon} />))}
+      </section>
```

Comment:
Kindly make sure that when the Pokemons are displayed on the homepage they have alternating background colors (one lighter background and the other darker background) as per design specifications.

![image](https://user-images.githubusercontent.com/38088818/205446843-94cf4a93-5d27-4254-ad8c-9f6d4e3ab60c.png)

![image](https://user-images.githubusercontent.com/38088818/205446917-f27cd585-c55a-49ce-8047-dc2b27801ce2.png)